ViewLayer/DashboardUI/I_DashboardUI.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <QLabel>
+#include <QTextEdit>
+
+class I_DashboardUI
```

Comment:
The UI files were named improperly before, all class names should not be camel case, even for abbreviations like UI. So the class name should be `I_DashboardUi`